src/csharp/Microsoft.Spark/Sql/RuntimeConfig.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Spark.Interop.Ipc;
+
+namespace Microsoft.Spark.Sql
+{
+    /// <summary>
+    /// Runtime configuration interface for Spark.
+    /// </summary>
+    public sealed class RuntimeConfig : IJvmObjectReferenceProvider
+    {
+        private readonly JvmObjectReference _jvmObject;
+
+        internal RuntimeConfig(JvmObjectReference jvmObject) => _jvmObject = jvmObject;
+
+        JvmObjectReference IJvmObjectReferenceProvider.Reference => _jvmObject;
+
+        /// <summary>
+        /// Sets the given Spark runtime configuration property.
+        /// </summary>
+        /// <param name="key">Config name</param>
+        /// <param name="value">Config value</param>
+        public void Set(string key, string value) => _jvmObject.Invoke("set", key, value);
+
+        /// <summary>
+        /// Sets the given Spark runtime configuration property.
+        /// </summary>
+        /// <param name="key">Config name</param>
+        /// <param name="value">Config value</param>
+        public void Set(string key, bool value) => _jvmObject.Invoke("set", key, value);
+
+        /// <summary>
+        /// Sets the given Spark runtime configuration property.
+        /// </summary>
+        /// <param name="key">Config name</param>
+        /// <param name="value">Config value</param>
+        public void Set(string key, long value) => _jvmObject.Invoke("set", key, value);
+
+        /// <summary>
+        /// Returns the value of Spark runtime configuration property for the given key.
+        /// </summary>
+        /// <param name="key">Key to use</param>
+        /// <param name="defaultValue">Default value to use</param>
+        public string Get(string key, string defaultValue) =>
```

Comment:
Let's also expose `public string Get(string key)`.

Reply:
I was considering this, but this will spark will throw a `java.util.NoSuchElementException` if the key does not exist.  This exact exception doesn't seem like it'll be propagated back through the `_jvmObject.Invoke` call.  I assume this was also why `SparkConf.cs` didn't implement this method.

Reply:
We have many APIs where exceptions are thrown on the JVM side. I also, check the PySpark and they expose this function as well.

Reply:
Added.